nuget/Autofac.Extras.MvvmCross.nuspec

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The core Autofac library does this. https://github.com/autofac/Autofac/blob/develop/src/Autofac/project.json